### PR TITLE
BUG: ValueError in restoration.denoise_bilateral for zeros image

### PR DESCRIPTION
**Pull request by evanlimanto**
Fix issue #1677 by checking if image.min() == image.max(). If so, return the image without denoising.

### CODE REVIEW

**Comment by jni** [comment]
@evanlimanto Nice! One suggestion: why not check in the Python wrapper? That is usually where we put this kind of preprocessing.

https://github.com/scikit-image/scikit-image/blob/master/skimage/restoration/_denoise.py#L78

Thanks!

**Comment by evanlimanto** [comment]
Thanks for the comment; I fixed the tests associated with this too.

**Comment by soupault** [comment]
@jni If you take charge of this PR, could you also check one more https://github.com/scikit-image/scikit-image/pull/1679 ?

File: skimage/restoration/tests/test_denoise.py [hunk 1]
```diff
@@ -172,6 +172,16 @@ def test_denoise_bilateral_2d():
...
 
 
+def test_denoise_bilateral_zeros():
+    img = np.zeros((10, 10))
+    assert_equal(img, restoration.denoise_bilateral(img))
+
+
+def test_denoise_bilateral_constant():
+    img = np.ones((10, 10)) * 5
+    assert_equal(img, restoration.denoise_bilateral(img))
```
Comment (jni) [inline skimage/restoration/tests/test_denoise.py]: Oops, I missed this the first time: it looks like this call needs a `multichannel=False` kwarg, or the test fails with a `ValueError`.

File: skimage/restoration/tests/test_denoise.py [hunk 2]
```diff
@@ -200,7 +210,7 @@ def test_denoise_bilateral_3d_multichannel():
         result = restoration.denoise_bilateral(img, multichannel=True)
 
     expected = np.empty_like(img)
-    expected.fill(np.nan)
+    expected.fill(1)
```
Comment (jni) [inline skimage/restoration/tests/test_denoise.py]: Maybe just remove this and assert_equal(result, img)?

**Comment by evanlimanto** [comment]
Fixed!

**Comment by evanlimanto** [comment]
Updated commit to implement @stefanv's comment.

**Comment by sciunto** [comment]
Thnak you @evanlimanto

**Pull request merged**